.npmignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
code coverage tests are failing I think because it can't find these tests - are you leaving them out for another PR?

Reply:
yes, they're ready to push whenever the next milestone opens on upwork. I've got to 100% coverage